Song/VerseWriter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class VerseWriter
+    {
+        public string WriteVerse(List<string> animals, List<string> animalLines)
+        {
+            string separator = animals.Count == 1 ? "." : ";";
+            string secondSeparator;
+            string firstLine = $"There was an old lady who swallowed a {animals[animals.Count - 1]}{separator}{Environment.NewLine}";
+            string secondLine = "";
```

Comment:
It's bad practice to define variables at the top. To improve readability, define a variable as close as it is used. In this case, if you do so, you will realise that secondSeparator can be a loop variable and not a function variable.